cfme/tests/control/test_alerts.py

```diff
@@ -185,6 +186,24 @@ def vm_crud(provider, vm_name, full_template):
     return VM.factory(vm_name, provider, template_name=full_template["name"])
 
 
+@pytest.fixture(scope="module")
+def ssh_appliance():
+    return SSHClient(
+        username=credentials['ssh']['username'],
+        password=credentials['ssh']['password'],
+        hostname=urlparse(store.base_url).netloc
```

Comment:
@quarckster where are we with this one?  It seems the ball is in your court, yes?